docs/user_guide/sqlserver/collection_scripts.md

```diff
@@ -92,44 +92,44 @@ In order to begin running the Database Migration Assessment Collection process,
 #### Perfmon Requirements (Optional)
 
 - If you have your own perfmon counters capturing the following statistics or run on a SQL Server Product such as Amazon RDS or Google CloudSQL for SQL Server, skip to step b, otherwise proceed to step a.
-** The Perfmon data collection process is optional and can be safely skipped.  However, there will be no right sizing information in the assessment report.
+  \*\* The Perfmon data collection process is optional and can be safely skipped. However, there will be no right sizing information in the assessment report.
   <br/>
```

Comment:
Is the HTML break supposed to be here?

Reply:
Yes.  Needed to add it once I checked the markdown in vscode.